src/scores_ui.cpp

```diff
@@ -55,126 +150,171 @@ static std::string get_achievements_text( const achievements_tracker &achievemen
-        achievements,
-        conducts,
-        scores,
-        kills,
-        num_tabs,
-        first_tab = achievements,
-    };
-
-    tab_mode tab = static_cast<tab_mode>( 0 );
-    input_context ctxt( "SCORES" );
-    ctxt.register_cardinal();
-    ctxt.register_action( "QUIT" );
-    ctxt.register_action( "PREV_TAB" );
-    ctxt.register_action( "NEXT_TAB" );
-    ctxt.register_action( "HELP_KEYBINDINGS" );
+    const kill_tracker &kills_data = g->get_kill_tracker();
 
-    catacurses::window w_view;
-    scrolling_text_view view( w_view );
-    view.set_up_navigation( ctxt, scrolling_key_scheme::arrow_scroll, true );
-
-    bool new_tab = true;
-
-    ui_adaptor ui;
-    const auto &init_windows = [&]( ui_adaptor & ui ) {
-        w = new_centered_win( TERMY - 2, FULL_SCREEN_WIDTH );
-        w_view = catacurses::newwin( getmaxy( w ) - 4, getmaxx( w ) - 1,
-                                     point( getbegx( w ), getbegy( w ) + 3 ) );
-        ui.position_from_window( w );
-    };
-    ui.on_screen_resize( init_windows );
-    // initialize explicitly here since w_view is used before first redraw
-    init_windows( ui );
-
-    const std::vector<std::pair<tab_mode, std::string>> tabs = {
-        { tab_mode::achievements, _( "ACHIEVEMENTS" ) },
-        { tab_mode::conducts, _( "CONDUCTS" ) },
-        { tab_mode::scores, _( "SCORES" ) },
-        { tab_mode::kills, _( "KILLS" ) },
-    };
-
-    ui.on_redraw( [&]( const ui_adaptor & ) {
-        werase( w );
-        draw_tabs( w, tabs, tab );
-        draw_border_below_tabs( w );
-        wnoutrefresh( w );
-
-        view.draw( c_white );
-    } );
+    const int monster_kills = kills_data.monster_kill_count();
+    const int npc_kills = kills_data.npc_kill_count();
 
-    while( true ) {
-        if( new_tab ) {
-            switch( tab ) {
-                case tab_mode::achievements:
-                    view.set_text( get_achievements_text( achievements, false, getmaxx( w ) - 2 ) );
-                    break;
-                case tab_mode::conducts:
-                    view.set_text( get_achievements_text( achievements, true, getmaxx( w ) - 2 ) );
-                    break;
-                case tab_mode::scores:
-                    view.set_text( get_scores_text( stats ) );
-                    break;
-                case tab_mode::kills:
-                    view.set_text( kills.get_kills_text() );
-                    break;
-                case tab_mode::num_tabs:
-                    cata_fatal( "Invalid tab" );
-                    break;
+    if( get_option<bool>( "STATS_THROUGH_KILLS" ) &&
+        ImGui::CollapsingHeader( _( "Stats through kills:" ), ImGuiTreeNodeFlags_DefaultOpen ) ) {
+        const int total_kills = kills_data.total_kill_count();
+        ImGui::TextWrapped( string_format( _( "Total kills: %d" ), total_kills ).c_str() );
+        ImGui::NewLine();
+        ImGui::TextWrapped( string_format( _( "Experience: %d (%d points available)" ),
```

Comment:
No need to call `string_format` here, as `ImGui::TextWrapped` can handle the formatting on its own.